src/nvim/testdir/Makefile

```diff
@@ -35,6 +34,9 @@ SCRIPTS ?= $(SCRIPTS_DEFAULT)
 # Tests using runtest.vim.
 # Keep test_alot*.res as the last one, sort the others.
 NEW_TESTS ?= \
+	    test_alot.res \
+	    test_alot_latin.res \
+	    test_alot_utf8.res \
```

Comment:
The "alot" tests are an exception to the "alphanumeric order" rule. If in doubt, just check Vim's order to see the expected order.